src/sentry/hybridcloud/rpc/services/caching/service.py

```diff
@@ -51,13 +40,13 @@ class SiloCacheBackedCallable(Generic[_R]):
     silo_mode: SiloMode
     base_key: str
     cb: Callable[[int], _R]
-    type: Type[_R]
+    type_: type[_R]
```

Comment:
here's the manual change (otherwise `type` accesses in annotations in methods refer to this local!)